### PR TITLE
SPP-9816-create-thousands-pounds-user-docs

### PR DESCRIPTION
**Pull request by joepercival**
The Thousand Pounds method currently has no supporting user documentation in the sml-supporting-info repository. This PR updates the user documentation to include guidance on how to use the Thousand Pounds method.

### CODE REVIEW

**Review by gibbardsteve — COMMENTED** [review]
This needs updating to reflect the latest changes submitted to main. Target_variable class has been renamed as TargetVariable.

I think we need to also share this with Ellie (once Sanjeev's latest CSV changes are made) to get her thoughts.

**Comment by sanjeevz3009** [comment]
Replying to Steve's comment above: My example.py/ CSV changes are in =)

File: method-info/thousand-pound-correction/thousand_pound_correction.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+- You will have to create a new python file importing in the `pandas_wrapper.py`.
+- Where you will have to write functions to read a CSV file and pass in the data as a DataFrame into the *`wrapper`* function from the `pandas_wrapper.py` file.
+
+
+We have an example of how to do this in the `pandas_example.py` file within the `utils` directory.
+
+
+### Additional Examples
+
+For more examples of using this method with CSV files or by implementing the pandas package see the totals_and_components source code [documentation page](https://github.com/ONSdigital/sml-python-small/blob/main/sml_small/editing/thousand_pounds).
```
Comment (sanjeevz3009) [inline method-info/thousand-pound-correction/thousand_pound_correction.md:139]: I think you forgot to replace `totals_and_components` source code with `thousand_pounds` source code =)

**Review by sanjeevz3009 — CHANGES_REQUESTED** [review]

File: method-info/thousand-pound-correction/thousand_pound_correction.md (hunk 1 above)
Comment (joepercival) [reply method-info/thousand-pound-correction/thousand_pound_correction.md]: Corrected now

File: method-info/thousand-pound-correction/thousand_pound_correction.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+## Test Data
+
+The [test data mentioned in the example above](https://github.com/ONSdigital/sml-supporting-info/blob/main/method-info/thousand_pounds/example-data) can be found alongside this user documentation.
```
[the remark this answers is not in the document]
Comment (joepercival) [reply method-info/thousand-pound-correction/thousand_pound_correction.md]: corrected link now

File: method-info/thousand-pound-correction/thousand_pound_correction.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+Once you have downloaded the sml-small wheel package you will be able to import the Thousand Pound Correction method and pass in data.
+
+The TPC markers returned determine if and what the method has corrected.
+
+* "S" is when the method stops early or the method is not completed
+* "N" is for no correction
+* "C" is for correction, where a correction has been applied to the target variables
+
+The core mathematical correction applied to selected target variables is shown below.
```
Comment (elliewatkins) [inline method-info/thousand-pound-correction/thousand_pound_correction.md:19]: Applied to the principal variable and selected target variables and

File: method-info/thousand-pound-correction/thousand_pound_correction.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+Example output table:
+
+| unique_identifier | principal_final_value | target_variables | tpc_ratio | tpc_marker |
+| --- | --- | --- | --- | --- |
+| '12340000001-201409-q100' | '5.0E+4' | [TargetVariable(identifier='q101', original_value='500', final_value='0.5'), TargetVariable(identifier='q102', original_value='1000', final_value='1'), TargetVariable(identifier='q103', original_value='1500', final_value='1.5'), TargetVariable(identifier='q104', original_value=None, final_value=None)] | '8.3E+2' | 'C' |
+
+Output attributes:
+* unique_identifier - Unique identifier e.g. a question code/ruref/period/id/combination of all of these
+* principal_final_value – Output value that may or may not be adjusted
+* target_variables - adjusted identifier/value pairs (returned as TargetVariable objects)
```
Comment (elliewatkins) [inline method-info/thousand-pound-correction/thousand_pound_correction.md:103]: target variables may not be adjusted, same as principal

**Review by elliewatkins — CHANGES_REQUESTED** [review]
Thanks for doing this, just a few small wording changes around the principal and target variables. Please see Pete's email for further info

File: method-info/thousand-pound-correction/thousand_pound_correction.md (hunk 4 above)
Comment (joepercival) [reply method-info/thousand-pound-correction/thousand_pound_correction.md]: Made the changes, thanks!

File: method-info/thousand-pound-correction/thousand_pound_correction.md (hunk 1 above)
Comment (sanjeevz3009) [reply method-info/thousand-pound-correction/thousand_pound_correction.md]: Thanks =) All good now!

**Review by sanjeevz3009 — APPROVED** [review]
Thank you for making the changes I requested =) Happy to approve on my side!

**Review by elliewatkins — APPROVED** [review]

**Pull request merged**